### PR TITLE
[SYCL] Add simple abs(int) to imf libdevice

### PR DESCRIPTION
**Pull request by jinge90**
This PR aims to add a simple "int32_t abs(int32_t)" to imf libdevice. The request is from DL framework developers.

### CODE REVIEW

**Comment by jinge90** [comment]
Hi, @zettai-reido and @akolesov-intel
Could you take a look at this PR?
Thanks very much.

File: libdevice/imf/imf_inline_fp32.cpp [hunk 1]
```diff
@@ -131,6 +131,10 @@ DEVICE_EXTERN_C_INLINE float __devicelib_imf_fminf(float a, float b) {
   return __fmin(a, b);
 }
 
+DEVICE_EXTERN_C_INLINE int32_t __devicelib_imf_abs(int32_t x) {
+  return (x > 0) ? x : -x;
```
Comment (zettai-reido) [inline libdevice/imf/imf_inline_fp32.cpp:135]: `>= 0`

**Review by zettai-reido — APPROVED** [review]
NIT: Usually, abs is defined >= 0 to be x.

File: libdevice/imf/imf_inline_fp32.cpp (hunk 1 above)
Comment (jinge90) [reply libdevice/imf/imf_inline_fp32.cpp]: Updated.
Thanks very much.

**Review by AlexeySachkov — APPROVED** [review]
`sycl-post-link` changes LGTM

**Review by akolesov-intel — APPROVED** [review]
Looks good to me

**Review by cperkinsintel — APPROVED** [review]

**Comment by jinge90** [comment]
Hi, @intel/llvm-gatekeepers 
Could you help merge this PR?
Thanks very much.

**Comment by jinge90** [comment]
Hi, @intel/llvm-gatekeepers 
Kind ping~

**Pull request merged**